sg_jira/bridge.py

```diff
@@ -73,6 +67,12 @@ def __init__(
         """
         super(Bridge, self).__init__()
 
+        # The bridge webserver is multithreaded, which means we need to
+        # track Shotgun connections via the API per thread. The SG Python
+        # API is not threadsafe, and using a single, global connection
+        # across all threads will lead to some weird behavior.
+        self._SG_CACHED_CONNECTIONS = threading.local()
```

Comment:
This was moved from the outside the init to inside, so that the variable was not static.
It was causing the tests to fail since the mocked Shotgun instance was being reused between tests.

Bridge appears to be instantiated once during normal use by the web server, so I don't think this should have any impact, and indeed does appear to work fine.